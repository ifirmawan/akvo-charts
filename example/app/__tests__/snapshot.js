import { render, waitFor } from '@testing-library/react';
import Home from '../page';
import ChartContextProvider from '../../context/ChartContextProvider';
import DisplayContextProvider from '../../context/DisplayContextProvider';

it('renders homepage unchanged', async () => {
  const { container } = render(
    <DisplayContextProvider>
      <ChartContextProvider>
        <Home />
      </ChartContextProvider>
    </DisplayContextProvider>
  );
  await waitFor(() => {
    expect(container).toMatchSnapshot();
  });
});
