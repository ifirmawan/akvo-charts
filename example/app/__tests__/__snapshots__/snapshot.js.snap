// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="w-full flex flex-col lg:flex-row gap-0 overflow-y-hidden"
  >
    <div
      class="w-full h-[calc(100vh-20px)] bg-white space-y-3 border-r border-zinc-300"
    >
      <div
        class="_styles-module__container__1Lxpd"
        role="figure"
      />
    </div>
  </div>
</div>
`;
